plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator.go

```diff
@@ -25,15 +25,14 @@ import (
 
 // HelmChartInflationGeneratorPlugin is a plugin to generate resources
 // from a remote or local helm chart.
-type HelmChartInflationGeneratorPlugin struct {
+type plugin struct {
```

Comment:
The pluginator convention is that this must be named `plugin` specifically. The `var KustomizePlugin` exclusion depends on it, and so that var was incorrectly being copied into the API module for the helm plugin.